### PR TITLE
Add `acll (class_name)` to print class information ##anal

### PR DESCRIPTION
**Pull request by HoundThe**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

I've added option to print "acll" information about single class

```
[0x00001120]> acl
Bat
Bird
Cat
Dog
Kappa
Mammal
[0x00001120]> acll Bat
Bat
  (vtable at 0x3b98)
  (vtable at 0x3bc0)
  virtual_0 @ 0x192e (vtable + 0x0)
  virtual_8 @ 0x1967 (vtable + 0x8)
  virtual_16 @ 0x146c (vtable + 0x10)
[0x00001120]> ac?
Usage: ac  anal classes commands
| acl[j*]                                                    list all classes
| acll (class_name)                                        list all or single class detailed
```

### CODE REVIEW

**Comment by XVilka** [comment]
Could you please add the JSON variant of this command too?

File: libr/core/cmd_anal.c [hunk 1]
```diff
@@ -9824,6 +9825,19 @@ static void cmd_anal_class_vtable(RCore *core, const char *input) {
...
+		if (input[1] == 'l') { // "acll" (name)
+			const char *arg = r_str_trim_head_ro (input + 2);
+			if (*arg) { // if there is an argument
+				char *class_name = strdup (arg);
+				if (!class_name) {
+					break;
+				}
+				char *name_end = (char *)r_str_trim_head_wp (class_name);
+				*name_end = 0; // trim the whitespace around the name
+				r_anal_class_print (core->anal, class_name, true);
```
Comment (trufae) [inline libr/core/cmd_anal.c:9837]: leakjing class_name and name_end

File: libr/anal/class.c [hunk 2]
```diff
@@ -937,7 +937,7 @@ R_API RAnalClassErr r_anal_class_vtable_delete(RAnal *anal, const char *class_na
 // ---- PRINT ----
 
 
-static void r_anal_class_print(RAnal *anal, const char *class_name, bool lng) {
+R_API void r_anal_class_print(RAnal *anal, const char *class_name, bool lng) {
```
Comment (trufae) [inline libr/anal/class.c:940]: find a better name for the `lng` parameter

File: libr/core/cmd_anal.c (hunk 1 above)
Comment (HoundThe) [reply libr/core/cmd_anal.c:9837]: name_end isn't allocation, so only class_name

File: libr/anal/class.c (hunk 2 above)
Comment (HoundThe) [reply libr/anal/class.c:940]: Maybe "detailed" or smth along these lines?

**Comment by HoundThe** [comment]
> Could you please add the JSON variant of this command too?

Even tho the original aclj prints out all the information in json already? Would that be useful to have in json for a single class?

**Comment by XVilka** [comment]
Yes, it will be useful for a single class as well, especially for Cutter folks.

**Pull request merged**